### PR TITLE
added getItem to retrieve item details from chrome web store

### PR DESCRIPTION
**Pull request by kalehrishi**
Added support for get Item from chrome web store

https://developer.chrome.com/webstore/webstore_api/items/get

### CODE REVIEW

**Comment by kalehrishi** [comment]
@pastak I didn't get chance to test this code yet. However it should work. Could you test it and merge if its working?

File: src/index.js [hunk 1]
```diff
@@ -84,6 +84,26 @@ program
...
   })
 
+
+program
+  .command('get [itemId]')
+  .description('make item publish')
+  .option('-t, --token [YOUR_TOKEN]', 'Your token')
+  .option('--projection [PROJECTION_TYPE]', '"DRAFT" or "PUBLISHED"')
+  .action(function (itemId, options) {
+    var token = getToken(options)
```
Comment (pastak) [inline src/index.js]: please use `const` instead of `var`

File: src/libs/chrome-webstore.js [hunk 2]
```diff
@@ -30,6 +30,15 @@ module.exports = class ChromeWebStore {
       body: fileBin
     })
   }
+  getItem (token, itemId, projection = "DRAFT") {
+    return request.get({
+      uri: 'https://www.googleapis.com//chromewebstore/v1.1/items/'+ itemId + "?projection="+projection,
```
Comment (pastak) [inline src/libs/chrome-webstore.js]: `standard` says you should use `'` but `"`

**Comment by pastak** [comment]
@kalehrishi Sorry for this repo has no test :cry: 

Please lint by standard.

OK. I'll test and if I check it works I'll merge it.

**Comment by kalehrishi** [comment]
@pastak I've added changes as per your review. Could you please test it.

**Comment by pastak** [comment]
LGTM

**Comment by kalehrishi** [comment]
:+1:

**Comment by kalehrishi** [comment]
@pastak Could you create a new release? Latest changes are not available via npm.

**Comment by pastak** [comment]
@kalehrishi Sorry for fogetting... 😿 

I released `v0.4.0`. Please check it !

**Comment by kalehrishi** [comment]
@pastak Thanks. I will check and update the status.

**Comment by markdreyer** [comment]
Deployed to WebStore Version 1.5.86

**Pull request merged**